scripts/auditdocgen/main.go

```diff
@@ -124,6 +124,12 @@ func updateAuditDoc(doc []byte, auditableResourcesMap AuditableResourcesMap) ([]
 		}
 		auditActionsString := strings.Join(auditActions, ", ")
 
+		// AuditableGroup is really a combination of Group and GroupMember resources
+		// but we use the label 'Group' in our docs to avoid confusion.
+		if resourceName == "AuditableGroup" {
+			resourceName = "Group"
+		}
```

Comment:
I expect this will still appear as `AuditableGroup` in the API/logs? 🤔 

I could see this still leading to confusion

Reply:
@bpmct here is the API response
````
{
    "id": "37e56b87-d7d3-42db-95f8-62e10dedd71c",
    "request_id": "ec79e5ac-6a52-4e23-9c40-d106b8a98214",
    "time": "2023-01-27T15:31:10.638635Z",
    "organization_id": "00000000-0000-0000-0000-000000000000",
    "ip": "72.227.97.34",
    "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/109.0.0.0 Safari/537.36",
    "resource_type": "group",
    "resource_id": "831fc5b9-d65b-44e9-b09c-43b22571853e",
    "resource_target": "KiraDemoGroup",
    "resource_icon": "",
    "action": "write",
    "diff": {
        "members": {
            "old": [],
            "new": [
                {
                    "group_id": "831fc5b9-d65b-44e9-b09c-43b22571853e",
                    "user_id": "1c3e3fff-6a0e-4179-9ba3-27f5443e6fce"
                }
            ],
            "secret": false
        }
    },
    "status_code": 200,
    "additional_fields": {},
    "description": "{user} updated group {target}",
    "resource_link": "",
    "is_deleted": false,
    "user": {
        "id": "1c3e3fff-6a0e-4179-9ba3-27f5443e6fce",
        "username": "Kira-Pilot",
        "email": "kira.pilot23@gmail.com",
        "created_at": "2022-08-10T16:11:50.602541Z",
        "last_seen_at": "0001-01-01T00:00:00Z",
        "status": "active",
        "organization_ids": null,
        "roles": [
            {
                "name": "user-admin",
                "display_name": "User Admin"
            },
            {
                "name": "template-admin",
                "display_name": "Template Admin"
            },
            {
                "name": "auditor",
                "display_name": "Auditor"
            },
            {
                "name": "owner",
                "display_name": "Owner"
            }
        ],
        "avatar_url": "https://avatars.githubusercontent.com/u/19142439?v=4"
    }
}
````

You can see that `resource_type` is `Group` and there is no mention of `AuditableGroup`, so I think we're okay but let me know if I'm missing something.

If you're wondering why I couldn't just rename it throughout the repo, I wish I could, but since we already have `database.Group` and that identifies a different model, I think this is the easiest solution.

Reply:
Ah perfect!

Reply:
Yeah, that definitely answers my question. We're good to go :)